### PR TITLE
[Upstream] UPDATES, UPDATES EVERYWHERE

### PR DESCRIPTION
**Pull request by Bobbanz1**
## About The Pull Request
Pulls the following PRs from Upstream:
* https://github.com/BeeStation/BeeStation-Hornet/pull/8414
* https://github.com/BeeStation/BeeStation-Hornet/pull/8406
* https://github.com/BeeStation/BeeStation-Hornet/pull/8304
* https://github.com/BeeStation/BeeStation-Hornet/pull/8398
* https://github.com/BeeStation/BeeStation-Hornet/pull/8358
* https://github.com/BeeStation/BeeStation-Hornet/pull/8278
* https://github.com/BeeStation/BeeStation-Hornet/pull/8437
* https://github.com/BeeStation/BeeStation-Hornet/pull/8436
* https://github.com/BeeStation/BeeStation-Hornet/pull/8439
* https://github.com/BeeStation/BeeStation-Hornet/pull/8438
* https://github.com/BeeStation/BeeStation-Hornet/pull/8444
* https://github.com/BeeStation/BeeStation-Hornet/pull/8440
* https://github.com/BeeStation/BeeStation-Hornet/pull/8405

## Why It's Good For The Game
UPDATES
UPDATES EVERYWHERE!!

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

nedladdning
</details>

## Changelog
:cl:EvilDragonfiend, Ivniinvi, Zytolg, RKz, CydiaLamiales, Shiraizawa, itsmeow
admin: welding/unwelding an airlock will be logged
code: changed a long var declaration from a single line into multiple lines for better readability
code: timers are put in the closest future bucket, as a precaution
tweak: medibots have more skins based on which first aid kit was used when it's made. These currently have no speicla ability by its shape which means these are just skins.
imageadd: more medibot skins (advanced, rad, syndi, doctors roundstart kit)
tweak: Phobias can now be weighted
tweak: The phobias of Robots and Doctors are slightly less likely to occur
tweak: The phobia of Space is significantly less likely to occur
add: Lizard speech now has a variation on the use of X in normal conversation. Things like "Xenomorph" will sound the same as before as it is phonetically similiar, however uses of "x-ray" will become "ecks-ray".
add: Added a line of text showing the Security Level in the current round
fix: fixed anomalies spawning inside walls
tweak: Instead of being dusted, plasmaman helmets now have their lamps broken when hit with the Light Eater
add: Added the ability to repair broken plasmaman helmet lamps with light bulbs
fix: Tying and untying bandanas will give the proper notice to people seeing it
balance: Decreased the snowball effect of fear build-up from (stress x10)% to (stress x5)%. Your stress levels build up faster with each fainting now, which leads to a higher chance of a heartattack
fix: You wont faint immediately after waking up from a faint caused by a phobia
fix: The message for fainting from a phobia will actually be visible for the one fainting now
add: Added a small bit of text indicating AIs should Ahelp before using the "Wipe-Core" verb
fix: Engineer goggles wont give meson lighting effects in their off state anymore.
tweak: Prevented multiple Space Pirates spawning in a round.
/:cl:

### CODE REVIEW

**Comment by Bobbanz1** [comment]
> I have to say, I prefer PRs to be one feature each, whenever possible. I find a clean commit history to be beneficial when reading code since good tooling (like Gitlens) allows you to easily see which commit a change came from. Since we're squashing PRs into one commit the commit message is the PR description. If a PR changes multiple things at once it is much less clear for what reason a piece of code was changed.

Hmmm, I could split the commits up into separate ones inside the PR if ya want?

**Comment by Kenionatus** [comment]
> Hmmm, I could split the commits up into separate ones inside the PR if ya want?

That would only be visible on Github and not become part of the git history (since PRs are squashed into single commits).

File: code/modules/clothing/spacesuits/plasmamen.dm [hunk 1]
```diff
@@ -210,6 +219,20 @@
 	user.update_inv_head() //So the mob overlay updates
 	update_button_icons(user)
 
+/obj/item/clothing/head/helmet/space/plasmaman/proc/smash_headlamp()
+	if(!lamp_functional)
+		return
+	if(!helmet_on)
+		return
```
Comment (Bokkiewokkie) [inline code/modules/clothing/spacesuits/plasmamen.dm:226]: This shouldn't be in this proc, it is already done before calling smash_headlamp() in the relevant cases.

**Review by Bokkiewokkie — APPROVED** [review]

**Comment by Tsar-Salat** [comment]
> Tsar-Salat, Ivniinvi, Zytolg, RKz,

Bro I already told you Tsar-Salat isnt a real person! 
😢

**Review by covertcorvid — APPROVED** [review]

**Pull request merged**